packages/common/src/scopeSupportFacets/scopeSupportFacets.types.ts

```diff
@@ -46,6 +46,7 @@ const scopeSupportFacets = [
   "branch.if",
   "branch.if.iteration",
   "branch.try",
+  "branch.try.iteration",
```

Comment:
I originally left this one out because I couldn't see how it would be useful. But I'm not opposed